src/chart/sankey/SankeyView.ts

```diff
@@ -250,7 +250,8 @@ class SankeyView extends ChartView {
                     width: layout.dx,
                     height: layout.dy
                 },
-                style: itemModel.getModel('itemStyle').getItemStyle()
+                style: itemModel.getModel('itemStyle').getItemStyle(),
+                z2: 10
```

Comment:
@pissang @susiwen8  Sorry here. I ever reverted it in #14542. But for now, this change seems to be still needed?

Reply:
@plainheart Seems so, do you mind adding it before we vote for this release?

Reply:
Sure. It's here. #14640